plugins/CorePluginsAdmin/Controller.php

```diff
@@ -247,10 +247,19 @@ protected function getPluginsInfo($themesOnly = false)
                     $suffix = "You may uninstall the plugin or manually delete the files in piwik/plugins/$pluginName/";
                 }
 
-                $description = '<strong>'
-                    . $this->translator->translate('CorePluginsAdmin_PluginNotCompatibleWith', array($pluginName, self::getPiwikVersion()))
-                    . '</strong><br/>'
-                    . $suffix;
+                if (is_dir(PIWIK_INCLUDE_PATH . '/plugins/' . $pluginName)) {
```

Comment:
Actually there is already a method in plugin manager. So will change it to use that one instead.